### PR TITLE
change color for different typeface and preprocessor

### PR DESCRIPTION
**Pull request by paingheinthu**

### CODE REVIEW

**Comment by mgrbyte** [comment]
@paingheinthu Please fix the conflicts then I'll look at merging.
Thanks.

**Comment by paingheinthu** [comment]
ahh okie, I miss out conflicts repo.

**Comment by paingheinthu** [comment]
hi bro, finish solve conficts.

File: abyss-theme.el [hunk 1]
```diff
@@ -36,7 +36,8 @@
        (abyss/magenta "#ff00ff")
        (abyss/hilite "#dd5542")
        (abyss/white "#ffffff")
-       (abyss/green "#00ff00"))
+       (abyss/green "#00ff00")
+	   (abyss/springgreen "#00fa9a"))
```
Comment (mgrbyte) [inline abyss-theme.el]: Fix indentation pls!

**Comment by mgrbyte** [comment]
@paingheinthu If you could provide a screen-shot of what this PR does to the theme, that'd be most appropriated too.

**Comment by mgrbyte** [comment]
@paingheinthu appreciated not appropriated! (sorry...)

**Comment by paingheinthu** [comment]
Hi @mgrbyte , I push one screenshot with C lang. please verify it is ok or not. Thank

**Comment by mgrbyte** [comment]
@paingheinthu 
I've checked out your branch and tested locally.
The screenshot looked great, but I develop a lot in clojure and python, and this changes
all the import/require names to be spinggreen instead of skyblue, so
whilst the code is fine, I'm afraid I really don't like the colour change.

Please feel free to fork if you don't like the colours!

**Comment by paingheinthu** [comment]
@mgrbyte  thank for test my changes. I need to clear colors with type face and pre processor in my C develop. so I was change this color for different.

**Pull request closed without merging**